src/main/resources/db/migration/V1_35__location.sql

```diff
@@ -0,0 +1,13 @@
+drop table location_mapping;
+
+create table location_mapping (
```

Comment:
I was looking at this for incidents aswell, wondering if we should have an index on the nomis id too?  None of the other tables do, but our main search is usually on the nomis value...what do you think?

Reply:
v good point yes, deff needs an index! Will add